src/install_rules.ml

```diff
@@ -287,22 +292,62 @@ module Gen(P : Params) = struct
        >>>
        Build.write_file_dyn fn)
 
-  let init_install () =
-    let entries_per_package =
-      List.concat_map (SC.stanzas_to_consider_for_install sctx)
+  let preinit =
```

Comment:
I find the following more descriptive:

```suggestion
  let bin_entries, other_stanzas =
```

And I suppose `init_install` can just take `()` as argument and access these directly.

Reply:
Indeed! That remained as an artifact of one of the intermediate changes.